test/unit/network_spec.js

```diff
@@ -176,4 +176,62 @@ describe('network', function() {
       done.fail(reason);
     });
   });
+
+  describe('PDFNetworkStreamFullRequestReader_extractNameFrom', function () {
+    var stream = new PDFNetworkStream({
+      source: {
+        url: pdf1,
+        rangeChunkSize: 65536,
+        disableStream: true,
+      },
+      disableRange: true,
+    });
+
+    var requestReader = stream.getFullReader();
+
+    it('returns null when no content disposition', function() {
+      expect(requestReader._extractNameFrom(null)).toBeNull();
+      expect(requestReader._extractNameFrom(undefined)).toBeNull();
+    });
+
+    it('Response header for the main body', function () {
```

Comment:
`gets the filename from the response header from the main body`. The same applies to the descriptions below, i.e., try to make the descriptions flow better with the code for readability.

Reply:
consider it done